sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/planner/BeamRuleSets.java

```diff
@@ -39,22 +60,91 @@
  * RelNode} tree, to represent with {@link BeamRelNode}
  */
 public class BeamRuleSets {
+  public static final RuleSet LOGICAL_OPT_RULES =
```

Comment:
Maybe do we want to have `BeamRuleSets.BEAM_CONVERTER_RULES` and `BeamRuleSets.LOGICAL_OPT_RULES`?

Then `getRuleSets()` can return them all for use with `Frameworks` and the `BeamLogicalConvention` use of rule sets can just register the converters.

Reply:
Any rules you want to add probably need to go into `getRuleSets` so they are applied in both the JDBC and non-JDBC path.

Reply:
done.